pennylane/utils.py

```diff
@@ -467,9 +467,6 @@ def get_generator(op, return_matrix=False):
     if return_matrix:
         obs = obs.matrix()
 
-        if not isinstance(obs, np.ndarray):
-            obs = obs.toarray()
-
```

Comment:
Since SparseMatrix returns dense matrices now, this is obsolete. And it is untested, so Codecov complains.